packages/foam-vscode/src/test/suite.ts

```diff
@@ -9,42 +9,43 @@
  *   and so on..
  */
 
+process.env.FORCE_COLOR = '1';
+process.env.NODE_ENV = 'test';
+
 import path from 'path';
 import { runCLI } from '@jest/core';
+import { cleanWorkspace } from './test-utils-vscode';
 
-const rootDir = path.resolve(__dirname, '../..');
+const rootDir = path.join(__dirname, '../..');
 
 export function run(): Promise<void> {
   const errWrite = process.stderr.write;
+
+  let remaining = '';
   process.stderr.write = (buffer: string) => {
-    console.log(buffer);
+    const lines = (remaining + buffer).split('\n');
+    remaining = lines.pop() as string;
+    // Trim long lines because some uninformative code dumps will flood the
+    // console or, worse, be suppressed altogether because of their size.
+    lines.forEach(l => console.log(l.substr(0, 300)));
     return true;
```

Comment:
This triggered a number of related reports:

* https://github.com/microsoft/vscode-test/issues/114
* https://github.com/microsoft/vscode-test/issues/115
* https://github.com/microsoft/vscode-test/issues/116
* https://github.com/microsoft/vscode/issues/139417
* https://github.com/foambubble/foam/issues/880

Reply:
What do you mean by triggered?

Reply:
These are all reports of mine, some of them related to what caused this change, some of them to what was caused by this change. My feeling is that vscode tooling is still somewhat of work in progress in this regard.